src/components/editor/document-bar/menus/export/markdown.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Dropdown } from 'react-bootstrap'
+import { download } from '../../../../common/download/download'
+import { ForkAwesomeIcon } from '../../../../common/fork-awesome/fork-awesome-icon'
+
+export interface MarkdownExportProps {
+  title: string
+  noteContent: string
+}
+
+export const MarkdownExport: React.FC<MarkdownExportProps> = ({ title, noteContent }) => {
```

Comment:
Isn't this more of a "MarkdownExportDropdownItem" ?

Reply:
it could be

Reply:
changed